modules/diagnostic-bundle-data.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * configuration/generate-diagnostic-bundle.adoc
+:_module-type: CONCEPT
+[id="diagnostic-bundle-data_{context}"]
+= Diagnostic bundle data
+
+When you generate a diagnostic bundle, it includes the following data:
+
+* Central heap profile
+* System logs: Logs of all {product-title} components (for the last 20 minutes) and logs of recently crashed components (from up to 20 minutes before the crash).
+System logs depend on the size of your environment.
+For large deployments, we collect log files for components with critical errors (for example, a high restart count) only.
```

Comment:
* Rewrite to avoid "we"
* Also maybe suggest to get rid of the parentheses (since there are a lot in this paragraph). Perhaps by "... with critical errors only, such as a high restart count."